client/galaxy/scripts/viz/circster.js

```diff
@@ -634,10 +634,10 @@ var CircsterTrackView = Backbone.View.extend({
             track.get('config').set_value('max_value', self.data_bounds[1], {silent: true});
 
             // Merge chroms layout with data.
-            layout_and_data = _.zip(chrom_arcs, genome_wide_data),
+            var layout_and_data = _.zip(chrom_arcs, genome_wide_data);
 
             // Render each chromosome's data.
-            chroms_data_layout = _.map(layout_and_data, function(chrom_info) {
+            _.each(layout_and_data, function(chrom_info) {
```

Comment:
The array this used to map into was never used anywhere, hence I removed it and replaced `_.map` with `_.each`. If this function is, in fact, side effect-free (which I'm not in a position to judge), then the entire call can be removed.